m4/ld-version-script.m4

```diff
@@ -0,0 +1,53 @@
+# ld-version-script.m4 serial 3
+dnl Copyright (C) 2008-2014 Free Software Foundation, Inc.
+dnl This file is free software; the Free Software Foundation
+dnl gives unlimited permission to copy and/or distribute it,
+dnl with or without modifications, as long as this notice is preserved.
+
```

Comment:
Is this compatible with GPL2?

Reply:
IANAL, but according to Fedora, yes: https://fedoraproject.org/wiki/Licensing/FSF_Unlimited_License

Reply:
@flihp @martinezjavier any thoughts?

Reply:
@tstruk I'm not sure why you are asking for GPLv2 compatibility since the tpm2-tss project is licensed under BSD.

But yes, as @drbild said I believe that's OK to include this macro since the FSFUL is a very permissive license and so BSD compatible (I'm not a lawyer though and license compliance is always confusing for me).

Reply:
@martinezjavier sorry I meant BSD. Thanks for input.